src/domains/ecommerce-app/api/ecommerce-io/v2/_openapi.json.tpl

```diff
@@ -773,6 +781,74 @@
           }
         }
       }
+    },
+    "/users/{userId}/stats": {
+      "get": {
+        "operationId": "getUserStats",
+        "description": "Retrieve stats about a user's usage of the pagoPA platform. Currently retrieves the last used payment method.",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "userId",
```

Comment:
the client does not know this info, it is the backend that must inherit it from the token jwt

Reply:
Did it while you were posting :-)